build.gradle

```diff
@@ -71,6 +72,8 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
 
+    compileOnly group: "com.google.errorprone", name: "error_prone_annotations", version: "2.3.4"
+    errorprone group: "com.google.errorprone", name: "error_prone_core", version: "2.3.4"
```

Comment:
are these versions correct? are the guarded annotations and the dependencies on different versioning systems?